### PR TITLE
Task 3 linter

### PR DESCRIPTION
**Pull request by suffergram**
Task 3 Linter: added eslint with airbnb config

### CODE REVIEW

File: src/Button/Button.jsx [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-        </button>
-    );
-}
\ No newline at end of file
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const Button = React.forwardRef(({ children = '╳', ...delegatedProps }, ref) => (
```
Comment (souljja) [inline src/Button/Button.jsx]: You can remove = '╳' because you already have it in defaultProps

File: src/Card/Card.jsx [hunk 2]
```diff
@@ -1,25 +1,72 @@
...
+import PropTypes from 'prop-types';
+import React from 'react';
+import Popover from '../Popover/Popover';
+
+import './style.css';
+
+function Card({
+  onMovieModalOpen, onDeleteModalOpen, onMovieInfoOpen, movie,
+}) {
+  const ref = React.useRef();
```
Comment (souljja) [inline src/Card/Card.jsx]: This ref is redundant. Please, remove it.

File: src/Button/Button.jsx (hunk 1 above)
Comment (suffergram) [reply src/Button/Button.jsx]: Removed it.

File: src/Card/Card.jsx (hunk 2 above)
Comment (suffergram) [reply src/Card/Card.jsx]: Removed the variable and put its content to the ref attribute.

File: src/Card/Card.jsx [hunk 3]
```diff
@@ -1,25 +1,70 @@
...
+  onMovieModalOpen, onDeleteModalOpen, onMovieInfoOpen, movie,
+}) {
+  const handleCardClick = () => {
+    onMovieInfoOpen(movie);
+  };
+
+  return (
+    <div
+      className="card"
+      ref={React.createRef()}
```
Comment (souljja) [inline src/Card/Card.jsx]: You don't use this ref, so it can be removed.

Comment (suffergram) [reply src/Card/Card.jsx]: Removed.

**Review by souljja — APPROVED** [review]

**Pull request merged**